env.ini

```diff
@@ -0,0 +1,2 @@
+USER_TOKEN=b7OrQHosXm
+API_TOKEN=wYC6mh9K3m
```

Comment:
#### Automated PR Comment From coverity
<!-- 6fe419abff8d5bf7dcd5d85056dfaf35 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Hard-coded secret
_Low CWE-798_
A secret, such as a password, cryptographic key, or token is stored in plaintext directly in the source code, in an application's properties, or configuration file. Users with access to the secret may then use the secret to access resources that they otherwise would not have access to. Secret type: Access token.
### How to fix
Avoid setting sensitive configuration values as string literals. Instead, these values should be set using variables with the sensitive data loaded from an encrypted file or a secret store.